dist/recipe.nsi

```diff
@@ -23,7 +23,11 @@
 ;---------------------------------
 ;General
  
-  OutFile "pkg/collector_sidecar_installer_${VERSION}_x64.exe"
+  !ifndef win32
+    OutFile "pkg/collector_sidecar_installer_${VERSION}_x64.exe"
+  !else
+    OutFile "pkg/collector_sidecar_installer_${VERSION}_x32.exe"
+  !endif
```

Comment:
The 32bit installer is currently called `_i386` what is more consistent with common architecture names imo.